### DISCUSSION TITLE
Atom m5 Pinout - Docs updating?

### DISCUSSION THREAD

**Discussion by daaaaan** [omerfaruk-aran/esphome_samsung_hvac_bus#118] [General]
Firstly thanks for this project. I've just come on board. 

A slight gotcha for the atom pinout as they seem to differ from the example.yaml 

TXD = G26 and RXD = G32 on some boards if you're using the tail + atom from pihut. 

https://thepihut.com/products/atom-tail485

![tempImagevFNm8w](https://github.com/lanwin/esphome_samsung_ac/assets/4655275/bdabe548-7881-46e2-b546-adff2631988d)

Thanks!

**Comment by lanwin** [comment]
@daaaaan dank you for your feedback. I extended the example.yaml.
